### DISCUSSION TITLE
[doc] incorrect description of vertexCount field in loaded 3D Tile content

### DISCUSSION THREAD

**Discussion by andreasplesch** [visgl/loaders.gl#3106] [General]
The documentation at
https://github.com/visgl/loaders.gl/blob/5f4973a1f740a56661fc0a08065237998d0b0eb6/docs/specifications/category-3d-tiles.md?plain=1#L102
has a copy and paste bug for the vertexCount field's description of its content. The description is  copied over from the modelMatrix field which of course does not apply.
An appropriate description may read instead
`Number of vertices in the loaded tile`
I file this here under Discussions since the issue template did not cover documentation bugs.

**Comment by ibgreen** [comment]
@andreasplesch Thanks for reporting. We can look into adding something for documentation bugs, but feel free to file future issues under the module in question and explain in the free text.

**Reply by andreasplesch** [reply]
Thanks, will do.
